aten/src/ATen/native/metal/MetalTensor.mm

```diff
@@ -17,7 +17,7 @@ class API_AVAILABLE(ios(10.0), macos(10.13)) MetalTensor::Impl {
         _numel(std::accumulate(
             std::begin(_sizes),
             std::end(_sizes),
-            1,
+            (int64_t)1,
```

Comment:
Can this not use the new function?

Reply:
Didn't want to add includes to mobile mostly, who knows what's going to break.

Reply:
I don't know about mobile, so sure?